WordPress/src/main/java/org/wordpress/android/ui/engagement/EngagementUtils.kt

```diff
@@ -31,4 +33,15 @@ class EngagementUtils @Inject constructor() {
             )
         }
     }
+
+    fun likesToTrainOfFaces(
+        likes: List<LikeModel>
+    ): List<TrainOfFacesItem> {
+        return likes.map { likeData ->
+            FaceItem(
+                    userId = likeData.likerId,
+                    userAvatarUrl = likeData.likerAvatarUrl!!
+            )
+        }
+    }
```

Comment:
Suggestion: Super nitpick, feel free to ignore 😁.  
We might want to [omit explicit param declaration](https://kotlinlang.org/docs/coding-conventions.html#lambda-parameters)  in lambda with no nesting and use `it` convention, reducing few lines of code.

```suggestion
    fun likesToTrainOfFaces(likes: List<LikeModel>) =
            likes.map { FaceItem(userId = it.likerId, userAvatarUrl = it.likerAvatarUrl!!) }
```